prac_08/dynamic_labels.py

```diff
@@ -0,0 +1,31 @@
+
+from kivy.app import App
+from kivy.lang import Builder
+from kivy.uix.label import Label
+
+
+class DynamicLabelsApp(App):
+    def __init__(self, **kwargs):
+        """Create main app"""
+        super().__init__(**kwargs)
+        self.names = ["Brodie", "Kevin", "Collins", "Chris"]
+
+    def build(self):
+        """Build the Kivy GUI."""
+        self.title = "Dynamic Labels"
+        self.root = Builder.load_file('dynamic_labels.kv')
+        return self.root
+
+    def on_start(self):
+        """create label"""
+        self.create_labels()
+
+    def create_labels(self):
+        """loop the list to display names"""
+        main_layout = self.root.ids.main
+        for name in self.names:
+            temp_label = Label(text=name)
+            main_layout.add_widget(temp_label)
```

Comment:
would the program be faster if u didnt define main_layout and just put it in the final line?